packages/itwinui-css/src/stepper/stepper.scss

```diff
@@ -82,6 +83,10 @@
   background-color: var(--_iui-stepper-step-background-color);
   color: var(--_iui-stepper-step-number-color);
 
+  :is(svg, img) {
```

Comment:
Why is `img` included here? `fill` does nothing on `img`.

Reply:
I have removed `img`.